plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterActive.php

```diff
@@ -48,142 +48,37 @@ protected function render( $attributes, $content, $block ) {
 		 */
 		$active_filters = apply_filters( 'collection_active_filters_data', array(), $this->get_filter_query_params( $query_id ) );
 
-		$wrapper_attributes = get_block_wrapper_attributes(
-			array(
-				'data-wc-interactive' => wp_json_encode( array( 'namespace' => $this->get_full_block_name() ), JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
-				'data-wc-key'         => 'product-filter-active-' . md5( wp_json_encode( $attributes ) ),
-			)
-		);
-
-		$list_classes = 'filter-list';
-
-		if ( 'chips' === $attributes['displayStyle'] ) {
-			$list_classes .= ' list-chips';
+		if ( empty( $active_filters ) ) {
+			return '';
```

Comment:
We can't do that as the Removable Chips style won't load initially, see pdnLyh-7qP-p2. We have to render all inner blocks of the Active block to have the style load correctly. You can look at Attribute block `hasFilterOptions` for inspiration.

Reply:
@dinhtungdu - ok I've pushed up a fix for this.